cmake/onnxruntime_server.cmake

```diff
@@ -24,6 +61,44 @@ endif()
 include(get_boost.cmake)
 set(re2_src ${REPO_ROOT}/cmake/external/re2)
 
+# Genrate GPRC stuff
```

Comment:
Typo and wording.

Reply:
I also pointed this out. @csteegz please don't resolve issues unless you actually address them

Reply:
yeah I accidentally reverted - I'll look over those again.